HW-1/task3.py

```diff
@@ -18,4 +18,4 @@
             print(f'Число {num} СОСТАВНОЕ')
             break
     if primary:
```

Comment:
Переменная primary  не нужна. Можно просто сделать так
```suggestion
    else:
```